plugins/brjs-check-i18n-plugin/src/main/java/org/bladerunnerjs/plugin/checki18n/CheckI18nCommand.java

```diff
@@ -143,17 +146,35 @@ private void generateAllTranslationsCSV(String appName) {
 		File file = brjs.storageFile(this.getClass().getSimpleName(), appName + "-tokens.csv" );
 		file.getParentFile().mkdirs();
 		
-		try(Writer writer = new BufferedWriter(new FileWriter(file));) {
+		try{
+			Writer writer = new BufferedWriter(new OutputStreamWriter(
+					new FileOutputStream(file), "UTF-8"));
+
 			writer.append("** the 'IsUsed' column only relates to tokens identified in their entirety and"
 					+ " will not include tokens which are concatenated e.g. i18n('a.token.' + value);\n");
 			for(List<String> row : rows){
 				writer.append(StringUtils.join(row, ",")+"\n");
-			}			
-		} catch (IOException e) {
-			e.printStackTrace();
+			}
+			writer.flush();
+			writer.close();
+
+			String content = FileUtils.readFileToString(file, "UTF-8");
+	        FileUtils.write(file, content, "UTF-8");
+			}
+			catch (UnsupportedEncodingException e) 
+			{
+				System.out.println(e.getMessage());
+			}
+			catch (IOException e)
+			{
+				System.out.println(e.getMessage());
+			}
+			catch (Exception e)
```

Comment:
yup. I'll address that when i fix the spec test which is failing, there must be a test that expects garbled encoding:b